web/src/user_pill.ts

```diff
@@ -120,7 +118,7 @@ export function get_user_ids(
     pill_widget: UserPillWidget | CombinedPillContainer | GroupSettingPillContainer,
 ): number[] {
     const items = pill_widget.items();
-    return items.flatMap((item) => (item.type === "user" ? (item.user_id ?? []) : [])); // be defensive about undefined users
```

Comment:
Can this line be part of the previous commit?

Reply:
Yeah, done!